src/webgl/loading.js

```diff
@@ -177,6 +177,94 @@ p5.prototype.loadModel = function(path) {
   }
   return model;
 };
+/**
+ * @method loadModel
+ * @param  {String} path
+ * @param  {Object} [options]
+ * @param  {function(p5.Geometry)} [options.successCallback]
+ * @param  {function(Event)} [options.failureCallback]
+ * @param  {String} [options.fileType]
+ * @param  {boolean} [options.normalize]
+ * @return {p5.Geometry} the <a href="#/p5.Geometry">p5.Geometry</a> object
+ */
+p5.prototype.loadModel = function(path,options) {
```

Comment:
It looks like we also still have the old implementation of `loadModel` above this. I think we would need to delete lines 106-176 so that the doc comments added here come directly after the last doc comment above the previous implementation.

Reply:
I removed the mentioned lines but the tests are failing.